web/test/pages/profile.test.tsx

```diff
@@ -2193,6 +2193,26 @@ describe("profile", () => {
     });
   });
 
+  it("resets the last visited page in form state when changing legal structure", async () => {
```

Comment:
I might say "user data" instead of "form state" because form state implies we're looking at the internal state of the form (ie, ProfileDataContext) but that's not what's under test